app/src/main/java/com/msaggik/playlistmaker/entity/Track.kt

```diff
@@ -1,11 +1,16 @@
 package com.msaggik.playlistmaker.entity
 
 import com.google.gson.annotations.SerializedName
+import java.io.Serializable
 
-data class Track(
+data class Track (
     @SerializedName("trackId")val trackId: Int, // Id композиции
     @SerializedName("trackName")val trackName: String, // Название композиции
     @SerializedName("artistName") val artistName: String, // Имя исполнителя
     @SerializedName("trackTimeMillis") val trackTimeMillis: Long, // Продолжительность трека
-    @SerializedName("artworkUrl100") val artworkUrl100: String // Ссылка на изображение обложки
-)
+    @SerializedName("artworkUrl100") val artworkUrl100: String, // Ссылка на изображение обложки
+    @SerializedName("collectionName") val collectionName: String, // Название альбома
+    @SerializedName("releaseDate") val releaseDate: String, // Год релиза
+    @SerializedName("primaryGenreName") val primaryGenreName: String, // Жанр трека
+    @SerializedName("country") val country: String // Страна исполнителя
```

Comment:
Лучше писать в таком формате 
```suggestion
    @SerializedName("country") 
    val country: String // Страна исполнителя
```

Reply:
Александр, приветствую, отлично, спасибо, к следующему спринту пофиксю

Reply:
По поводу бага с неотображением автора песни, абсолютно согласен, я как то пропустил этот момент, обязательно поправлю
Нажатие на один элемент списка также поправлю, согласен для пользователей это удобнее чем маленький кликер

Александр, спасибо огромное за подробную обратную связь, для меня это ценно